ZividOpenCV/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -0,0 +1,129 @@
+/*
+Undistort an RGB image from a ZDF point cloud using Zivid camera intrinsics.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <opencv2/calib3d/calib3d.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#include <iostream>
+
+void pointCloudToRGB(const Zivid::PointCloud &, cv::Mat &);
+void getCameraIntrinsics(const Zivid::Camera &, cv::Mat &, cv::Mat &);
+void displayRGB(const cv::Mat &, const std::string);
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to the camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        std::cout << "Configuring the camera settings" << std::endl;
+        camera << Zivid::Settings::Iris{ 21 } << Zivid::Settings::ExposureTime{ std::chrono::microseconds{ 20000 } }
+               << Zivid::Settings::Gain{ 1 } << Zivid::Settings::Brightness{ 1.0 };
+
+        std::cout << "Capturing a frame" << std::endl;
+        const auto frame = camera.capture();
+
+        const auto intrinsics = camera.intrinsics();
+
+        std::cout << "Setting up visualization" << std::endl;
+        Zivid::CloudVisualizer vis;
+        zivid.setDefaultComputeDevice(vis.computeDevice());
+
+        std::cout << "Displaying the point cloud" << std::endl;
+        vis.showMaximized();
+        vis.show(frame);
+        vis.resetToFit();
+
+        std::cout << "Running the visualizer. Blocking until the window closes" << std::endl;
+        vis.run();
+
+        std::cout << "Converting ZDF point cloud to OpenCV format" << std::endl;
+
+        const auto pointCloud = frame.getPointCloud();
+        cv::Mat rgb((int)pointCloud.height(), (int)pointCloud.width(), CV_8UC3, cv::Scalar(0, 0, 0));
```

Comment:
Avoid C-style casts, and use a `static_cast` instead.

C++ Core Guidelines: [ES.49: If you must use a cast, use a named cast](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es49-if-you-must-use-a-cast-use-a-named-cast)

Reply:
Done